### ISSUE TITLE
Python dependency for FITS file generation.

### ISSUE THREAD

**Issue by hbelatikar** [hbelatikar/H2FCommunicator#5]
After event data is collected, the system invokes the python script "convertFITS.py". This script also has dependencies on 'numpy' as well as 'astropy'. Code should be developed to check these settings and allow the user to install them during runtime if needed.

**Comment by abunickabhi** [comment]
I guess the code to check this setting whether the user's system has _numpy_ and _astropy_ or not, will be a shell script. 

In the first event data collection run, the shell script will have to be run in LabVIEW. Will have to figure it out.

**Comment by hbelatikar** [comment]
Yes! you can look up how I open .csv files using the command prompt to see how shell scripting is done. 
You can do this check when the program starts up. 
Also you can show a window prompt to the user stating that 'numpy or astropy is not installed so the fits file will not be generated'.
